stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/MessageListFragment.kt

```diff
@@ -92,10 +92,12 @@ public open class MessageListFragment : Fragment() {
     private var _binding: StreamUiFragmentMessageListBinding? = null
     protected val binding: StreamUiFragmentMessageListBinding get() = _binding!!
 
-    override fun onAttach(context: Context) {
```

Comment:
Why is this code removed?

Reply:
i have no idea to be honest, maybe i did by accident, but dont remember doing it, will revert it.